### ISSUE TITLE
Make VoiceRecordComposerTile overlay MessageComposer in narrow mode

### ISSUE THREAD

**Issue by germain-gg** [element-hq/element-web#18806]
### Your use case

Make the `VoiceRecordComposerTile` overlay `MessageComposer` in narrow mode.

As part of vector-im/element-web#18533 we have hidden the voice recording button to pursue more important value point regarding to threading

### Additional context

<img width="327" alt="Screen Shot 2021-08-27 at 09 47 49" src="https://user-images.githubusercontent.com/769871/131100125-974acb60-0263-471f-a95f-ce38546d8c8a.png">

Some visual exploration. The waveform width would be responsive

**Comment by masterflitzer** [comment]
why exactly was the voice recording button hidden? it's not quite clear to me how this is related to threading...

**Comment by t3chguy** [comment]
@masterflitzer because the component falls apart at narrow widths, threads are always narrow yet will want voice recording so the threads team will handle the fix.

**Comment by masterflitzer** [comment]
@t3chguy but this shouldn't affect normal UI, also it should be responsive to window size no?
so when I have my window in fullscreen I should be able to see the icons because there is enough space...

**Comment by t3chguy** [comment]
@masterflitzer this issue is about the voice message button being entirely missing in narrow mode

![image](https://user-images.githubusercontent.com/2403652/156526169-739a0c42-91d5-412f-92b4-f7d7bba54d8b.png)

See https://github.com/vector-im/element-web/issues/20887#issuecomment-1054100649

**Comment by masterflitzer** [comment]
thx for pointing me to the right issue, could you explain how i get into the narrow mode you are showing?

**Comment by t3chguy** [comment]
The one I showed is from threads, but just making your window narrow or your left&right panels wide (so that the timeline is less than 500px wide) will put you in narrow mode.

**Comment by masterflitzer** [comment]
ok the voice button is missing for me too, it happens as soon the attachment button gets hidden, maybe they forgot to add the voice button to the hidden menu when switching to narrow mode or there is a limit of 4 items that needs to be lifted

**Comment by t3chguy** [comment]
> ok the voice button is missing for me too, it happens as soon the attachment button gets hidden, maybe they forgot to add the voice button to the hidden menu when switching to narrow mode or there is a limit of 4 items that needs to be lifted

No. Its this entire issue, the UX for recording a voice message doesn't fit in such a narrow view, hence it was removed in that case and this issue was created to make it responsive and bring it back.
